src/Client.php

```diff
@@ -182,6 +196,22 @@ public function setDefaultChannel($channel)
     }
 
     /**
+     * @return bool
+     */
+    public function isStickyChannel()
+    {
+        return $this->sticky_channel;
```

Comment:
According to the phpdoc, `sticky_channel` must be a boolean and developpers should respect this.

Casting anything to a boolean is not the right way IMO.

When this project will use PHP >=7.0, then we will be able to use strict type instead.